freetype/plan.sh

```diff
@@ -10,14 +10,12 @@ pkg_filename=${pkg_name}-${pkg_version}.tar.gz
 pkg_shasum=371e707aa522acf5b15ce93f11183c725b8ed1ee8546d7b3af549863045863a2
 pkg_deps=(core/glibc core/libpng core/zlib core/bzip2)
 pkg_build_deps=(core/gcc core/make core/coreutils core/pkg-config core/diffutils)
-pkg_lib_dirs=(lib)
-pkg_include_dirs=(include)
 pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+pkg_pconfig_dirs=(lib/pkgconfig)
 
 do_build() {
```

Comment:
I think we can remove the do_build function completely for this one since it is now the default implementation